assets/src/styles/blocks/ENForm/components/_enform.scss

```diff
@@ -146,7 +146,7 @@
       margin-top: 19px;
       line-height: 1;
 
-      button {
+      button, div.btn {
```

Comment:
Is there a reason to include the `div` element? If not better to keep specificity as low as possible.